### ISSUE TITLE
ADA: News Template - Search button focus indicator (Major / Easy)

### ISSUE THREAD

**Issue by bbusenius** [uchicago-library/library_website#903]
**Parent Issue:** #332

## Problem
- The focus indicator (e.g. border or dotted underline / background etc.) is not visible as one tabs through the page.
- "search" button lacks visual focus indicator
- [Screenshot example](https://wsassure-reports.dequecloud.com/api/v1/file/3d4fe4a0-2d43-11ea-8f3b-176ee3b4f585)

## Related code
```html
<input type="submit" value="Search" class="searchbox-submit" style="background-color: transparent; color: transparent;">
```

## Solution
Add a focus indicator of some type. Common keyboard focus indicator solutions include:

1. A 2px box around the focused element
2. A change in the background color of the focused element
3. The addition of an icon, such as an arrow, next to a menu item
4. The addition of a thick line under or next to the focused element
5. A distinctive change to the text of the component such as making it bold and/or underlined
6. Use the default browser focus indicator. NOTE: Since each browser has its own default focus indicator style, check each of the major browsers (Chrome, Firefox, Edge, IE, Safari) to ensure the default focus indicator is visible as you tab through the page.

**Severity:** Major / Easy
